x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/container/QueryContainer.java

```diff
@@ -366,8 +366,8 @@ private FieldExtraction topHitFieldRef(FieldAttribute fieldAttr) {
     private Tuple<QueryContainer, FieldExtraction> nestedHitFieldRef(FieldAttribute attr) {
         String name = aliasName(attr);
         Query q = rewriteToContainNestedField(query, attr.source(),
-                attr.nestedParent().name(), name, 
-                SqlDataTypes.format(attr.field().getDataType()), 
+                attr.nestedParent().name(), name,
+                SqlDataTypes.format(attr.field().getDataType()),
```

Comment:
Please format only the touched lines - there's a plugin/setting for IDEA that does that.

Reply:
For those such changes that touch code lines, it adds some more effort when checking the git history, or simply "blame". On the other hand, personally I have no problem when deleting whitespaces empty lines, as long as it's in the touched files.